ansible/README.md

```diff
@@ -36,10 +36,13 @@ and deploying it onto your AWS instance.
 	git push origin pick-a-branch-name
 	```
 3) Login to your new AWS server, clone your docker-cloud-test repository.
+
+fml: ...from the ansible directory from your new branch...
+
 4) Run the configure-host.yml playbook (on the AWS server) as-is to verify ansible is setup.
 	```bash
 	# Replace xxxxxxx here with your AWS username
-	ansible-playbook /mnt/src/configure-host.yml -v --extra-vars "student_username=xxxxxxx"
```

Comment:
Ah, thanks.  I was testing in a local docker instance and was mounting the ansible folder into the container at `/mnt/src`.